client/revoke_all.go

```diff
@@ -0,0 +1,59 @@
+package client
+
+import (
+	"context"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/skupperproject/skupper/api/types"
+	"github.com/skupperproject/skupper/pkg/kube"
+)
+
+func (cli *VanClient) RevokeAccess(ctx context.Context) error {
+	records, err := cli.KubeClient.CoreV1().Secrets(cli.Namespace).List(metav1.ListOptions{LabelSelector: "skupper.io/type=token-claim-record"})
+	if err != nil {
+		return err
+	}
+	for _, record := range records.Items {
+		err = cli.KubeClient.CoreV1().Secrets(cli.Namespace).Delete(record.Name, nil)
+		if err != nil {
+			return err
+		}
+	}
+
+	ca, err := kube.RegenerateCertAuthority(types.SiteCaSecret, cli.Namespace, cli.KubeClient)
+	if err != nil {
+		return err
+	}
+	siteServerSecret := types.Credential{
+		Name:    types.SiteServerSecret,
+		Subject: types.TransportServiceName,
+	}
+	usingRoutes := false
+	if cli.RouteClient != nil {
+		route, err := kube.GetRoute(types.InterRouterRouteName, cli.Namespace, cli.RouteClient)
```

Comment:
Should the types.ClaimRedemptionRouteName route be included in regeneration?

Reply:
Yes. Should be fixed now.